src/test/kotlin/org/jetbrains/research/ml/ast/transformations/commentsRemoval/CommentsRemovalTransformationTest.kt

```diff
@@ -35,16 +40,4 @@ class CommentsRemovalTransformationTest : TransformationsTest(getResourcesRootPa
             getBackwardTransformationWrapper(CommentsRemovalTransformation::forwardApply)
         )
     }
-
-    @Test
-    fun testCommandStorage() {
```

Comment:
Why did you delete it?

Reply:
Because it does some command, undoes it, and then re-does it again. We cannot re-do the same delete command since it stores the exact psiElement that changes after undo, so for now, it's not possible to redo it